### PR TITLE
Removes genetics for real now

### LINKED ISSUE #4829: mapmerge bug in atmos
http://puu.sh/Avfne/6d6efac210.jpg

http://puu.sh/Avfnw/ba68a3416d.png

### PR DESCRIPTION
**Pull request by Alberyk**
-removes all genetics things from the map/rd
-fixes #4829
-fully removes the geneticist job

### CODE REVIEW

**Comment by Alberyk** [comment]
Yes, cargo.

**Comment by DovaKitty** [comment]
How do I delete another person's post?

**Comment by Kaedwuff** [comment]
What about if someone wants to update it later?

**Comment by LordFowl** [comment]
Then they can re-add it?

File: maps/aurora/aurora-4_mainlevel.dmm [hunk 1]
```diff
@@ -7169,22 +7169,6 @@
...
 /obj/effect/floor_decal/industrial/outline/yellow,
-/obj/machinery/door/window/southleft{
-	dir = 4;
-	icon_state = "left";
-	name = "Atmospheric Hardsuits";
-	req_access = list(24)
-	},
-/obj/structure/table/rack,
-/obj/item/clothing/suit/space/void/atmos,
-/obj/item/clothing/head/helmet/space/void/atmos,
```
Comment (LordFowl) [inline maps/aurora/aurora-4_mainlevel.dmm]: What are these changes?

Comment (Alberyk) [reply maps/aurora/aurora-4_mainlevel.dmm]: Fixing a mapping bug. The issue #4829.

Comment (Kaedwuff) [reply maps/aurora/aurora-4_mainlevel.dmm]: They're fixes to an atmos duplicate items mappping error, per #4829

Comment (LordFowl) [reply maps/aurora/aurora-4_mainlevel.dmm]: Very atomic.

**Comment by NonQueueingMatt** [comment]
thank god

**Pull request merged**